src/js/tech/html5.js

```diff
@@ -321,7 +321,7 @@ class Html5 extends Tech {
     if ('webkitDisplayingFullscreen' in video) {
       this.one('webkitbeginfullscreen', function() {
         this.one('webkitendfullscreen', function() {
-          this.trigger('fullscreenchange');
+          this.trigger('fullscreenchange', 'not-fullscreen');
```

Comment:
Yeah, `{ fullscreen: false }` is good for me. I was debating what to do for it specifically but then just did the string to get the PR out.
The main question is whether we want to do it this way or is there a better way of doing it.